app/AppKernel.php

```diff
@@ -58,7 +58,7 @@ public function registerBundles()
             new AntiMattr\Bundle\MongoDBMigrationsBundle\MongoDBMigrationsBundle(),
         );
 
-        if (in_array($this->getEnvironment(), array('dev', 'test'))) {
+        if (in_array($this->getEnvironment(), array('dev', 'test', 'load_test'))) {
```

Comment:
i think the debugBundle, WebProfiler and SensioDistribution are not useful for the load_test environnement